ci.bash

```diff
@@ -167,7 +167,13 @@ function test::report
 function test::capture
 {
     if which juju-crashdump; then
-        juju-crashdump -s -a debug-layer -a config -m "$JUJU_CONTROLLER:$JUJU_MODEL"
+        juju-crashdump \
+            -s                   # small crashdump by skipping /var/lib/juju \
+            -a debug-layer       # included debug-layer addon \
+            -a config            # included config addon \
+            -j snap.kube*        # included logs from all kube* daemons \
```

Comment:
Does juju-crashdump expand service names like `snap.kube*` automatically? I don't see it in the help output, but I'm guessing you tried it?

Reply:
in it i see things like:
```
Mar 20 19:30:12 juju-51e5ea-0 systemd[1]: snap.kubelet.daemon.service: Start request repeated too quickly.
...
Mar 20 19:30:18 juju-51e5ea-0 systemd[1]: snap.kube-proxy.daemon.service: Scheduled restart job, restart counter is at 5.
...
Mar 20 19:30:11 juju-51e5ea-0 systemd[1]: snap.kubelet.daemon.service: Scheduled restart job, restart counter is at 4.

```